lib/fs.js

```diff
@@ -255,11 +255,15 @@ Object.defineProperties(fs, {
 });
 
 function handleError(val, callback) {
+  if (typeof val === 'string')
```

Comment:
Would you mind explaining this change?

It looks like it means wrapping a lot of other calls with `if(typeof x !== 'string')`